### DISCUSSION TITLE
No way to update domain settings

### DISCUSSION THREAD

**Discussion by liamsegura** [vercel/community#6619] [Help]
### Summary

Once a domain is redirected, I cannot see an option to update a TXT or MX file?

### Steps to Reproduce

Change name servers and redirect to real domain from vercels. 
No option to add aditional settings once set

**Comment by amyegan** [comment]
Hey @liamsegura. Did you add the domain to your [team](https://vercel.com/dashboard/domains) or [account](https://vercel.com/account/domains) domains list?

https://vercel.com/guides/how-do-i-add-a-custom-domain-to-my-vercel-project
